recipes/ftgl/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set name = "ftgl" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: "https://github.com/frankheckenbach/ftgl/archive/v{{ version }}.tar.gz"
+  sha256: "aa97da1c3442a8fd3941037655df18016d70b5266381c81d81e8b5335f196ea8"
+  patches:
+    - cmake_fix_2.4.1.patch
+
+build:
+  number: 0
+  run_exports:
+    # Hasn't been updated in years so pin tightly
+    - {{ pin_subpackage('ftgl', max_pin='x.x.x') }}
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make  # [unix]
+    - ninja  # [win]
+    - cmake
+    # Required for OpenGL support
+    # See https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=cdt#libgl
+    - {{ cdt('mesa-libgl-devel') }}  # [linux]
+    - {{ cdt('mesa-dri-drivers') }}  # [linux]
+    - {{ cdt('libselinux') }}  # [linux]
+    - {{ cdt('libxdamage') }}  # [linux]
+    - {{ cdt('libxxf86vm') }}  # [linux]
+  host:
+    - xorg-libxfixes  # [linux]
+    - libglu  # [linux]
+    - freetype
+
+  run:
+    - libglu  # [linux]
+    - freetype
+            
+
+test:
+   commands:
+     - test -f $PREFIX/lib/libftgl$SHLIB_EXT  # [unix]
+     - if not exist %LIBRARY_LIB%\ftgl.lib exit 1  # [win]
+
+about:
+  home: https://github.com/frankheckenbach/ftgl
+  license: LGPL-2.0-or-later
+  license_family: MIT
```

Comment:
```suggestion
  license_family: LGPL
```

Reply:
Actually, this is MIT, I just need to fix the one above instead.